index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="style.css">
+    <title> my portfolio</title>
+</head>
+
+<body>
+    <header>
+        <a class="logo">logos</a>
+        <nav class="humbuger"><img src="images/icones/Union.png" alt="humburger"> </nav>
+    </header>
+    <section class="main-container">
+        <h1 class="item1">Faustin</h1>
+        <h2 class="item2">Software Developer</h2>
+        <p class="item3">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <hr/>
+        <div class="border-bottom">
```

Comment:
- [REQUESTED] Kindly as requested by the previous reviewer You should add the section separator line at the top of this section. You can use `<hr> `tag for that.
- [ ] You must stick to the design as much as possible (e.g. font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [ ] Kindly the page has a scrollbar, remove it and let see your project approved :handshake: 

Your Design
![Screenshot from 2022-05-27 23-08-39](https://user-images.githubusercontent.com/77842237/170783516-559c7413-9082-446c-801f-ab95221a33e1.png)

Reply:
hello! @vic778 , can you please check very well my latest commit. because it's totally different from this screenshot